readme.rst

```diff
@@ -110,7 +110,7 @@ Usage
     # Mongodb's console output
     tail -F /var/log/mongodb/mongod.log
 
-.. |build-status| image:: https://travis-ci.org/ImageMarkup/isic-archive.svg?branch=master
+.. |build-status| image:: https://travis-ci.org/ImageMarkup/isic-archive/master.svg
```

Comment:
Fixing. The issue before was that the badge wasn't displaying the status of "master", but just the most recent build.